buildkite/src/Jobs/Release/MinaArtifactBuster.dhall

```diff
@@ -0,0 +1,116 @@
+let Prelude = ../../External/Prelude.dhall
```

Comment:
can we combine MinaArtifactBuster and MinaArtifactStretch into one function that can handle both?

I would recommend making a helper file and having a function that looks something like:

```dhall
let DebVersion = < Buster | Stretch >

let capitalName = \(debVersion : DebVersion) ->
  merge { Buster = "Buster", Stretch = "Stretch" } debVersion
 
let lowerName = \(debVersion : DebVersion) ->
  merge { Buster = "buster", Stretch = "stretch" } debVersion


let pipeline = \(debVersion : DebVersion) ->
  Pipeline.build
    ...
    name = "MinaArtifact${capitalName(debVersion)}"
    ...
    step_key="daemon-devnet-${lowerName(debVersion)}-docker-image"
    ...
    
let buster = pipeline Buster
let stretch = pipeline Stretch
```

and then you can use this in the MinaArtifactBuster.dhall files and MinaArtifactStretch.dhall files with:

```dhall
let ArtifactPipelines = ../../path/to/AritfactHelper.dhall in
ArtifactPipelines.buster
```